### ISSUE TITLE
Markdown doesn't support italic in the middle of words

### ISSUE THREAD

**Issue by SamiSaves** [sstur/draft-js-utils#179]
Currently it is impossible to use italic in the middle of words, because the import plugin will render underscores in the middle of words as underscores instead of turning them into italic (As it should, see [commonmark specs](https://spec.commonmark.org/0.29/#emphasis-and-strong-emphasis) and [github specs](https://github.github.com/gfm/#emphasis-and-strong-emphasis)).

For example exporting "test*italic*word" and then importing it back would result in `test_italic_word`.

## Possible fix
I tried fixing it by changing the italic from underscore to asterisk inthe draft-js-export-markdown plugin and for me it seemed to fix the issue.

[In source](https://github.com/sstur/draft-js-utils/blob/master/packages/draft-js-export-markdown/src/stateToMarkdown.js#L236)

**Comment by sstur** [comment]
OK. I'd be open to a PR to use asterisk, but it's a breaking change so perhaps only do it when necessary to disambiguate an italic within a word. Other ideas to minimize breakage?

**Comment by SamiSaves** [comment]
Yeah it probably needs to be considered as a breaking change if someone trusts the plugin to generate underscores, but I doubt it will cause much breakage. Markdown should consider the asterisk and the underscore nearly identical, and the import plugin will work fine with both.

**Comment by krystofbe** [comment]
Just stumbled over this, any progress on this? Would you accept a PR changing the italic from underscore to asterisk in the draft-js-export-markdown ?

**Comment by sstur** [comment]
Yea, totally hit me with a PR. Would be happy to review!

[Edit] Both underscore and italics should work for italic in Markdown. If in doubt, do check how GH does it and let's try to mimic that behavior.

**Comment by fedeciancaglini** [comment]
Any update on this? I'm having the same issue.

**Comment by SamiSaves** [comment]
We decided to fork the repository and apply the fix in this pr manually https://github.com/sstur/draft-js-utils/pull/182

I never got any comments on PR so I never tried to fix the tests or tried to finalize it.

**Comment by Nic0S** [comment]
@sstur Any chance you could take a look at the PR? I know it's ancient that this point, I'd be happy to take ownership and make any fixes / write any tests necessary to get it merged
